### PR TITLE
[4.3][bug] fix 5.2 + debug lang

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #39651 .

### Testing Instructions
Check the issue

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by heelc29** [comment]
`warning` is also possible
https://github.com/joomla/joomla-cms/blob/92c014932228dae7a225184a04b01f0566b336c9/build/media_source/plg_system_debug/widgets/languageFiles/widget.es5.js#L20

```html
<tr>
    <td>pkg_en-GB.sys</td>
    <td>
        <ul>
            <li class="alert-warning">- /language/en-GB/pkg_en-GB.sys.ini</li>
            <li class="alert-warning">- /language/en-GB/en-GB.pkg_en-GB.sys.ini</li>
        </ul>
    </td>
</tr>
```

**Comment by heelc29** [comment]
`success` works
image

File: build/media_source/plg_system_debug/css/debug.css [hunk 1]
```diff
@@ -12,3 +12,15 @@
...
 }
+
+.phpdebugbar-widgets-languageFiles .alert-success {
```
Comment (heelc29) [inline build/media_source/plg_system_debug/css/debug.css]:
```suggestion
.phpdebugbar-widgets-languageFiles .alert-success,
.phpdebugbar-widgets-languageFiles .alert-warning {
```

File: build/media_source/plg_system_debug/css/debug.css [hunk 2]
```diff
@@ -12,3 +12,15 @@
...
+  color: var(--alert-color);
+  background-color: var(--alert-bg);
+  border-color: var(--alert-border-color);
+}
+
+.phpdebugbar-widgets-languageFiles .alert-warning {
+  color: var(--alert-warning);
+  background-color: var(--warning-bg);
+  border-color: var(--alert-border-color);
+}
```
Comment (heelc29) [inline build/media_source/plg_system_debug/css/debug.css]:
```suggestion
```

I think `--alert-warning` `--warning-bg` don't exist
image

Comment (dgrammatiko) [reply build/media_source/plg_system_debug/css/debug.css]: Oops copy pasting without thinking...

Thanks for catching this

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on d29ea2bcabb21f37330b37f5a63d1b9833d7f446<br /><br />image

Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d29ea2bcabb21f37330b37f5a63d1b9833d7f446<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by heelc29** [comment]
@dgrammatiko Should this code maybe added to `plg_system_debug/widgets/languageFiles/widget.css` instead of `plg_system_debug/css/debug.css`?

**Comment by dgrammatiko** [comment]
@heelc29 yeah that seems the right place. I already moved the code there, thanks

**Comment by heelc29** [comment]
I have tested this item :white_check_mark: successfully on 4ea06bb5e40ff69abe879f9257a6221e127c47a5<br /><br />Still works<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by richard67** [comment]
Back to pending. One more human test needed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4ea06bb5e40ff69abe879f9257a6221e127c47a5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39652">issues.joomla.org/tracker/joomla-cms/39652</a>.</sub>

**Comment by obuisard** [comment]
Thank you Dimitris @dgrammatiko for the fix!

**Pull request merged**